.circleci/config.yml

```diff
@@ -74,12 +74,15 @@ jobs:
 
             - run:
                 name: Run tests
-                command: npm run test.standalone
+                command: |
+                    rm -rf node_modules/cypress
+                    npm i cypress@3.4.1
+                    npm run test.standalone
```

Comment:
Updating Chrome from `71` to `78` and Cypress from `3.4.1` to 3.6.1` here seems to cause
- test performance degredation that accumulates leading to tests timing out eventually
- a different behavior that I can explain causing ~15% of tests to fail

While unit and server test suites are now running against latest, leaving the standalone tests as-is in its "legacy" state for now. I can't tell what's going on atm.

Reply:
Sure, I'm OK with this as long as we make an issue and come back to it in not too long - perhaps around Cypress 3.7?

Reply:
https://github.com/plotly/dash-table/issues/642